src/docs/documentation.ts

```diff
@@ -131,4 +131,33 @@
  *         description: User not found
  *       500:
  *         description: Error occurred while deleting user
+ * /api/v1/{userId}/update-password:
+ *   put:
+ *     summary: Change user password
+ *     tags: [Users]
+ *     requestBody:
+ *       description: Update an existing password
+ *       required: true
+ *       content:
+ *           application/json:
+ *               schema:
+ *                   type: object
+ *                   properties:
+ *                       userId:
+ *                           type: string
+ *                           description: User ID
+ *                       oldPassword:
+ *                           type: string
+ *                           description: Old password
+ *                       newPassword:
+ *                           type: string
+ *                           description: New password
+ *                       confirmPassword:
+ *                           type: string
+ *                           description: Confirm new password
+ *     responses:
+ *       '200':
+ *         description: Password updated successfully
+ *       '400':
+ *         description: Invalid request or old password
```

Comment:
We'll have documentation files specific to controllers. e.g: auth.docs.ts, user.docs.ts, etc...